stdlib/REPL/src/REPL.jl

```diff
@@ -1523,6 +1522,15 @@ function run_frontend(repl::StreamREPL, backend::REPLBackendRef)
     nothing
 end
 
+function load_pkg()
+    pkgid = Base.PkgId(Base.UUID("44cfe95a-1eb2-52ea-b672-e2afdf69b78f"), "Pkg")
```

Comment:
```suggestion
    Base.generating_output(true) && return nothing
    pkgid = Base.PkgId(Base.UUID("44cfe95a-1eb2-52ea-b672-e2afdf69b78f"), "Pkg")
```
to avoid the spurious dependency issue Jameson mentioned. Other than that, I think this is good to go.